src/components/HomepageFeatures/About.js

```diff
@@ -31,26 +31,26 @@ const About = () => {
               competition is the order of the day.
             </div>
             <div className='flex items-start justify-between w-full'>
-              <div className='flex flex-col items-start p-3 w-[45%] gap-2 bg-[#302261] rounded-xl'>
+              <div className='flex flex-col items-start p-3 w-[45%] gap-2 border border-[#60548A] bg-[#302261] rounded-xl'>
```

Comment:
It looks different than the design in figma 🤔

Reply:
nope. look closely. border exists